profile/README.md

```diff
@@ -0,0 +1,31 @@
+
+_API Integrations the right way_
+
+[![Discord](https://img.shields.io/discord/819563244418105354?logo=discord&logoColor=fff)](https://sfc.is/discord)
+[![NPM](https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2Fsuperfaceai)](https://twitter.com/superfaceai)
+
+
+<!-- not sure where to put this -->
+# What problem are we trying to solve?
+
+API integrations take forever to build. They’re brittle and hard to test. APIs can change on you without notice and break your integrations. Superface is trying to change this. If your Node.js app depends on multiple API integrations Superface is right tool for you!
+Our motivation behind Superface is nicely described in this [video](https://www.youtube.com/watch?v=BCvq3NXFb94) from APIdays conference.
+
+# Welcome to Superface
+
+Superface is a language and protocol for abstracting integrations to application use cases. It allows use case discovery and distribution of integration code at runtime.
+This approach gives you a framework to decouple lifecycle of your application and integrations it uses.
+
+
+## Technology behind
+
+<!-- One or two sentences then leading to superface.ai -->
+Superface technology consists of two parts: Registry which allows use case discovery and distribution of integration code at runtime and Client executing the integration code in your application. You can read more about our technology [here](https://superface.ai). For more details about Superface visit [how it works](https://superface.ai/how-it-works), [get started](https://superface.ai/docs/getting-started) or complete [documentation](https://superface.ai/docs).
```

Comment:
I think we should keep just one tutorial like page where to go. And not sure about getting started link and section Get started bellow.

```suggestion
Superface technology consists of two parts: Registry which allows use case discovery and distribution of integration code at runtime and Client executing the integration code in your application. You can read more about our technology [on Superface site](https://superface.ai) or to complete [documentation](https://superface.ai/docs).
```